ddl/reorg.go

```diff
@@ -149,7 +149,7 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, lease time.Dura
 		// Update a job's RowCount.
 		job.SetRowCount(rowCount)
 		// Update a reorgInfo's handle.
-		err := t.UpdateDDLReorgHandle(job, doneHandle)
+		err := t.UpdateDDLReorgStartHandle(job, doneHandle)
```

Comment:
Shall we update the endHandle and the partitionID herer?

Reply:
If we not, we adding partition id 2 indices, and we crashed, then the startHandle set to 1000, then after we restart the job, we will get previous partition id, and the incorrect startHandle.

Reply:
This code is compatible to the old one.
The `reorgInfo` will not change to another partition automatically, the only place to do it is in function `updateReorgInfo`.
I think update both the `endHandle` and the `partitionID` here would bring more uncertain things.